pages/index.js

```diff
@@ -97,6 +97,77 @@ const Home = () => (
           link="https://blog.commit.dev/articles/streaming-out-the-monolith"
         />
       </div>
+      <hr className={styles.sectionSeparator} />
+      <ol className={styles.timeline}>
```

Comment:
This could be its own section in the `/sections/` folder.

Reply:
Addressed